src/libpmem2/badblocks_ndctl.c

```diff
@@ -590,13 +590,13 @@ pmem2_badblock_context_delete(struct pmem2_badblock_context **bbctx)
 
 	ASSERTne(bbctx, NULL);
```

Comment:
I don't think we do this for any other public API function.
Delete functions must be called on properly initialized instances of the things they delete - otherwise, the behavior is unspecified - NULL is just one special case of "uninitialized".

Anyway, if you think this is neccessery, I think I'd prefer to simply do `if (*bbctx == NULL) return;`.